spec/migrations/20160414094300_change_capacity_to_hash_from_persistent_volume_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_migration
+
+describe ChangeCapacityToHashFromPersistentVolume do
+  let(:persistent_volume_stub) { migration_stub(:PersistentVolume) }
+
+  migration_context :up do
+    it "changes capacity to hash" do
+      persistent_volume_stub.create!(:capacity => 'storage=10Gi')
+      migrate
+      expect(persistent_volume_stub.first.capacity).to eq(:storage => 107_374_182_40)
```

Comment:
Prefer `10.gigabytes` over the numbers so the test is more readable and obvious.  (Same with the specs below)